vbfml/scripts/err_analysis

```diff
@@ -783,5 +826,66 @@ def plot_correl(ctx, input_dir: str) -> None:
         f.write(table_print)
 
 
+@cli.command()
+@click.pass_context
+@click.option(
+    "-i",
+    "--input-dir",
+    required=True,
+    help="path to the prediction directory in a model",
+)
+@click.option(
+    "-c", "--cut-variable", required=True, help="The quantity to cut on the threshold."
+)
+@click.option(
+    "-t", "--threshold", type=float, required=True, help="The threshold/cut value."
+)
+@click.option(
+    "-p",
+    "--plot-variable",
+    default="score",
+    required=True,
+    help="The variable to plot the distribution after the cut",
+)
+def dist_cut(
+    ctx, input_dir: str, cut_variable: str, threshold: float, plot_variable: str
+) -> None:
+    """
+    Apply a cut on the  cut-variable and plot the distribution of the plot-variable after the cut
+    """
+
+    # load data
+    features, predictions, _, labels = load_cache(input_dir)
+
+    # adapt variable if "score" is wanted, because "score" is not in feature
+    read_plot_variable= "mjj" if (plot_variable == "score") else plot_variable
+    read_cut_variable= "mjj" if (cut_variable == "score") else cut_variable
```

Comment:
Sorry I think I'm slightly confused why this is needed. Can't we do something like this so that we don't have to do this modification of variables:

```python
# One possible hack: Add scores to the features dictionary
features["score"] = predictions

# Once that is done, we can go ahead and construct our df with peace of mind
d = pd.DataFrame(
  {
    cut_variable: features[cut_variable],
    plot_variable: features[plot_variable],
    ...
  }
)
```